user/pages/04.Reference/08.network/docs.en.md

```diff
@@ -100,10 +100,13 @@ Currently our VPNaaS implementation supports the following algorithms in both ph
 
 | DH Groups      |
 | -------------- |
-| Group 2        |
 | Group 5        |
 | Group 14       |
 
+!! **Deprecation warning**
```

Comment:
Can we prioritize to remove it instead of documenting that it can be chose but is broken?

Reply:
The Merge Requests for disabling API/UI deprecated dh group2 settings are ready. From my point of view we can also re-work this part of the docs!

Reply:
Removed the part mentioning it is still chooseable.